pkgs/development/libraries/ncurses/default.nix

```
@@ -39,7 +39,7 @@ stdenv.mkDerivation (finalAttrs: {
     ++ lib.optionals stdenv.hostPlatform.isWindows [
       "--enable-sp-funcs"
       "--enable-term-driver"
-  ] ++ lib.optionals (stdenv.hostPlatform.isUnix && stdenv.hostPlatform.isStatic) [
+  ] ++ lib.optionals (stdenv.hostPlatform.isUnix && (enableStatic || stdenv.hostPlatform.isStatic)) [
```

I think it can be simplified to this:

```suggestion
  ] ++ lib.optionals (stdenv.hostPlatform.isUnix && enableStatic) [
```

I don't think the package would build if `stdenv.hostPlatform.isStatic == true && enableStatic == false`.

---

@midchildan Does `enableStatic` get activated automatically on `stdenv.hostPlatform.isStatic`?
Or do I need to change the default value for `enableStatic` to `stdenv.hostPlatform.isStatic`?

---

The default value for `enableStatic` already is `stdenv.hostPlatform.isStatic`, so no further changes should be necessary.

https://github.com/NixOS/nixpkgs/blob/742eb483001595f27be20c2eee501be8623150fe/pkgs/development/libraries/ncurses/default.nix#L8